src/Microsoft.AspNetCore.Razor.Language/Legacy/ParentChunkGenerator.cs

```diff
@@ -47,10 +47,7 @@ public override string ToString()
 
             public void Accept(ParserVisitor visitor, Block block)
             {
-                for (var i = 0; i < block.Children.Count; i++)
-                {
-                    block.Children[i].Accept(visitor);
-                }
```

Comment:
Is this just cleanup or was something wackyu going on?

Reply:
Nothing wacky but this was written in a way that was not overridable. This change lets me override this behavior in the serializer. This doesn't affect the product code behavior in any way.